.github/workflows/build.yml

```diff
@@ -20,11 +20,14 @@ jobs:
             fail-fast: false
             matrix:
                 php: ["8.0", "8.1"]
-                node: ["14.x"]
+                node: ["16.x"]
                 mysql: ["5.7", "8.0"]
                 symfony: ["^4.4", "^5.4"]
-                sylius: ["1.11.*"]
-
+                sylius: ["1.11.*", "~1.12.x-dev"]
```

Comment:
`~` before `1.12.x-dev` is a "hack" to satisfy `composer validate` as it was throwing
> require.sylius/sylius : exact version constraints (1.12.x-dev) should be avoided if the package follows semantic versioning